lib/assets/javascripts/cartodb/table/menu_modules/filters/templates/selection.jst.ejs

```diff
@@ -3,7 +3,7 @@
 <div class="header <%- status %>">
   <div class="title">
     <span>
-      <a class="toggleButton" style="float:left; margin-right:3px" href="#">(toggle)</a>
```

Comment:
How are the changes in this file related to not showing a pointer?

Reply:
Just cleaning up the code.  I moved the style from inline to the css sheet to better allow someone to make adjustments in the future.